src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -261,11 +259,25 @@ void WgRenderSettings::release(WgContext& context)
 
 void WgRenderDataPaint::release(WgContext& context)
 {
-    bindGroupPaint.release();
+    context.pipelines->layouts.releaseBindGroup(bindGroupPaint);
+    context.releaseBuffer(bufferModelMat);
+    context.releaseBuffer(bufferBlendSettings);
     clips.clear();
 };
 
 
+void WgRenderDataPaint::update(WgContext& context, const tvg::Matrix& transform, tvg::ColorSpace cs, uint8_t opacity)
+{
+    WgShaderTypeMat4x4f modelMat(transform);
+    WgShaderTypeBlendSettings blendSettings(cs, opacity);
+    if ((context.allocateBufferUniform(bufferModelMat, &modelMat, sizeof(modelMat))) |
```

Comment:
`|` -> `||` ?

Reply:
No, MUST be | instead of ||

Reply:
because you want to ensure both are called? can't you use && ?

Reply:
now we get warning, if it has to be `|` cast into int

Reply:
We need to chick if any of _context.allocateBufferUniform_ return false

Reply:
OK... I think I know how to fix warning properly. And this worning appears only on clang compiler. I will rewrite this logic properly